src/material/core/focus-indicators/structural-styles.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import {ChangeDetectionStrategy, Component, ViewEncapsulation} from '@angular/core';
+
```

Comment:
I feel like for the focus indicators the styles are basic enough that we can just bundle them in with each component instead of loading them dynamically.